blocks/top-table-list-block/features/top-table-list/default.scss

```diff
@@ -43,56 +43,27 @@
   .promo-container {
     display: flex;
     flex-direction: row;
-
-    > div {
-      &:nth-child(odd) {
-        padding-right: 0.75rem;
-      }
-
-      &:nth-child(even) {
-        padding-left: 0.75rem;
-      }
-    }
+    column-gap: calculateRem(24px);
 
     &.horizontal.reverse {
       flex-direction: row-reverse;
 
       > div {
         padding-left: 0;
         padding-right: 0;
-
-        &:nth-child(odd) {
```

Comment:
did we add this in somewhere? or has the style changed?

Reply:
Applied row class to parent and then used `column-gap`

Reply:
gotcha makes sense